### DISCUSSION TITLE
sidestore

### DISCUSSION THREAD

**Discussion by OptimusGREEN** [advplyr/audiobookshelf-app#1212] [Q&A]
Hi,

I don't see audiobookshelf listed in the ios sidestore as per the app faq. Do I have to add a repository or something?

**Comment by nichwall** [comment]
Yes, you will need to follow the directions for SideStore (at the time of writing, at the end of this page https://sidestore.io/)
You can use the IPA included in the most recent app release https://github.com/advplyr/audiobookshelf-app/releases

**Reply by OptimusGREEN** [reply]
Great,

Thank you for the quick reply.

**Comment by OptimusGREEN** [comment]
I checked those links and turns out I’ve already done that but don’t know how to find the app. If I have to add the abs repo I’m not sure what the URL is.
